components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/RustObject.kt

```diff
@@ -41,12 +41,14 @@ abstract class RustObject<T> : Closeable {
     }
 
     companion object {
+        const val RUST_STRING_ENCODING = "utf8";
```

Comment:
@thomcc Was about to merge this, but there is a lint error now because of the unneeded semicolon. Can you remove it?